tests/Integration/Database/Sqlite/SchemaStateTest.php

```diff
@@ -13,18 +18,16 @@ class SchemaStateTest extends DatabaseTestCase
         'database/schema',
     ];
 
+    #[RequiresOperatingSystem('Linux|Darwin')]
     public function testSchemaDumpOnSqlite()
     {
         if ($this->driver !== 'sqlite') {
             $this->markTestSkipped('Test requires a SQLite connection.');
         }
 
-        $connection = DB::connection('sqlite');
-        $connection->getSchemaBuilder()->createDatabase($connection->getConfig('database'));
+        UserFactory::new()->create();
```

Comment:
This is not equivalent to the database statements I had here previously, so now the `sqlite_stat*` tables aren't actually created and this test will always pass even if those tables aren't being removed from dumps correctly. I'll fix it in a new PR.

Reply:
Our GitHub Actions doesn't test using persistent SQLite and only using `:in-memory:`. so `DB::connection('sqlite')` is not the correct usage.

Reply:
I know, this test specifically needs a real database file. The dump file created by the test with your changes is empty, so it's not testing anything anymore.